suites/pacific/upgrades/tier-1_upgrade_test-4x-to-5x-rpm.yaml

```diff
@@ -113,13 +113,12 @@ tests:
           name: rbd-io
           module: rbd_faster_exports.py
           config:
-            io-total: 100M
-            cleanup: false
+            io-total: 30M
```

Comment:
Why reducing IO ?

Reply:
Hi @vamahaja 

With volume attached to the VM , changed IOs working as expected
Otherwise we are getting intermittent issues.